src/docs/asciidoc/reference/manage/organization.adoc

```diff
@@ -0,0 +1,36 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+An *organization* is a concept to let you work in team.
+
+The organization commonly groups all *ZetaPush accounts* of your company.
+You can have several applications for your organization and you can manage access rights on applications according to members of your team.
+
+For example, if you are in the company named "MyCompany" which has two applications named "MyApplication1" and "MyApplication2" :
+
+- One organization is created with the name "MyCompany"
+- Two applications are created and named "MyApplication1" and "MyApplication2"
+
+The *ZetaPush account* of this organization can access to these two applications. You can also change the access rights link:#access_control_per_application[by application] or link:#access_control_per_application[by environment].
```

Comment:
You talk about access control per environment but we never talked about it for now. Maybe we need an introduction in Manage ZetaPush that explains all concepts or maybe we should not talk about it for now and talk about it in manage applications